src/views/home/subviews/HomeTripSubview.module.css

```
@@ -497,6 +499,8 @@
   font-size: 14px;
   font-weight: 500;
   color: #000;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)